bin/barnard59.js

```diff
@@ -11,8 +11,6 @@ import { BatchSpanProcessor } from '@opentelemetry/tracing'
 
 import { Option, Command } from 'commander'
 
-diag.setLogger(new DiagConsoleLogger())
-
```

Comment:
Make optional because it tends to flood the standard output with warning message `Dropping extra events`

Reply:
I guess you could also do `diag.setLogger(new DiagConsoleLogger(), DiagLogLevel.ERROR);` instead, that particular log line is at the `warn` level

Reply:
So what about making this `--otel-debug <level>` with values matching the enum?